app/controllers/manageBusinesses/cease/DeclareIncomeSourceCeasedController.scala

```diff
@@ -139,9 +139,9 @@ class DeclareIncomeSourceCeasedController @Inject()(val authorisedFunctions: Fro
     if (isAgent) routes.CeaseIncomeSourceController.showAgent()
     else         routes.CeaseIncomeSourceController.show()
 
-  private val redirectAction: (Option[String], Boolean, IncomeSourceType) => Call = (id, isAgent, incomeSourceType) =>
-    if (isAgent) routes.IncomeSourceEndDateController.showAgent(id, incomeSourceType)
-    else         routes.IncomeSourceEndDateController.show(id, incomeSourceType)
+  private val redirectAction: (Option[String], Boolean, Boolean, IncomeSourceType) => Call = (id, isAgent, isChange, incomeSourceType) =>
+    if (isAgent) routes.IncomeSourceEndDateController.show(id, incomeSourceType, isAgent, isChange)
```

Comment:
if control flow statement is not required here, 
you can just call in one line:
`routes.IncomeSourceEndDateController.show(id, incomeSourceType, isAgent, isChange)`

Reply:
Made the corrections and removed statement from here